startup.d/10-git-pull.sh

```diff
@@ -16,6 +16,7 @@ fi
 
 # End code auditing section
 popd
+rm -r "$TEST_DIR";
```

Comment:
It will removed too. See https://github.com/chaosbot/chaos/pull/250/commits/a175ecb600b6a30ad8d69bcb1476f2b8f798c2ab#diff-8d12b14fde22f0fe54cea293ca1caa32R14 .